docs/get-started.md

```diff
@@ -0,0 +1,114 @@
+
+4. Click **Generate new token**.
+
+5. Give your token a descriptive name.
+
+6. Select scope `repo -> public_repo` (Access public repository)
+
+7. Click **Generate token** and copy your personal access token.
+
+8. Go to [travis-ci](https://travis-ci.com/) and login using your Github account
+
+9. Add access to your forked `bundle` repository.
+
+10. In the left sidebar, click **My Repositories** tab and choose your Github repository
+
+11. In the right tab click **More options** then **Settings**
+
+12. Go to **Environment Variables** section and add key-value pair where key is `GITHUB_TOKEN` and value is equal to your Github personal access token.
+
+13. In the right tab click **More options** then **Trigger build** and wait for Travis to finish his work
+
+### Create bundle
+
+1. Create your bundle according to [instruction](https://kyma-project.io/docs/latest/components/service-brokers#configuration-how-to-create-a-ybundle)
+
+2. Place your bundle into the `bundle/{repository}` directory.
+
+3. Along your bundle directory create `index.yaml` which defines available bundles. 
+Instruction how to create a valid file can be found [here](https://github.com/kyma-project/kyma/blob/master/docs/service-brokers/docs/011-configuration-helm-broker.md#configuring-the-helm-broker-externally)
+
+4. Commit changes.
+
+5. Travis automatically launches new build. You can check build status on [travis-ci](https://travis-ci.com/) webpage.
+
+### Setup Kyma Helm Config Map
+
+1. Go to your forked repository, click **Settings** tab, then go to **GitHub Pages** section and copy Github pages link next to **Your site is published at** phrase
+
+2. Before you change `helm-config-map` base url to your repository url, go to terminal enter
+    ```
+    kubectl describe cm helm-config-map -n kyma-system | awk '/baseURL:/ {print $2}'
+    ```
+    and copy previous configuration.
+    
+3. Setup env variable on Helm Broker Deployment
+    ```
+    > kubectl set env -n kyma-system deployment/core-helm-broker -e APP_REPOSITORY_BASE_URL=https://user-name.github.io/bundles/{repository}/
+    ```
+    > **NOTE:** To be sure you have this config map you can display all config maps `kubectl get cm -n kyma-system`
+
+4. Wait until kubernetes restart `core-helm-broker` Pod.
+
+    > **NOTE:** Check if Pod is running by `kubectl get pod -n kyma-system -l app=core-helm-broker` command
+
+5. Synchronize Service Catalog
+    ```
+    svcat sync broker core-helm-broker
+    ```
+
+### Check your Kyma Servise Catalog
+
+1. Go to Kyma console in your browser
+
+2. Choose Environment.
+
+3. In the left sidebar, click **Catalog**
+
+4. The new bundles should appear. If not then go to the Troubleshooting section.
+
+## Troubleshooting
+
+### Check Helm Broker logs
+
+First check pod's logs, it can give you a lot of information about occured issue. To check logs first find pod name
+in terminal enter:
+
+```
+kubectl get pods -n kyma-system | grep 'core-helm-broker' | grep -v 'etcd'
```

Comment:
you can simply use label selector
`kubectl logs -n kyma-system -l app=core-helm-broker`